### PR TITLE
docs: correct kubectl cmd for port forwarding

### PR DESCRIPTION
**Pull request by dahankzter**
The command to get the pod name for the port forwarding was incorrectly given.

Fixes: #129

### CODE REVIEW

File: docs/generic.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ Now the dashboards can be created along with the grafana plugin like this:
  To access Grafana locally, run:
  
  ```
- export POD_NAME=$(kubectl get pods --namespace monitoring -l "app=grafana,release=scylla-graf" -o jsonpath="{.items[0].metadata.name}")
+ export POD_NAME=$(kubectl get pods --namespace monitoring -l "app.kubernetes.io/instance=scylla-graf" -o jsonpath="{.items[0].metadata.name}")
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply docs/generic.md:278]: Sure but to what effect?

[the remark this answers is not in the document]
Comment (mmatczuk) [reply docs/generic.md:278]: I mean like `kubectl --namespace monitoring port-forward $(kubectl get pods --namespace monitoring -l "app.kubernetes.io/instance=scylla-graf" -o jsonpath="{.items[0].metadata.name}") 3000`

[the remark this answers is not in the document]
Comment (mmatczuk) [reply docs/generic.md:278]: also --namespace could be replaced with -n

[the remark this answers is not in the document]
Comment (dahankzter) [reply docs/generic.md:278]: Ah ok that looks nice, let's do that.

**Pull request merged**